doc/source/cluster/index.rst

```diff
@@ -115,11 +122,18 @@ should look something like ``123.45.67.89:6379``).
 If you wish to specify that a machine has 10 CPUs and 1 GPU, you can do this
 with the flags ``--num-cpus=10`` and ``--num-gpus=1``. See the :ref:`Configuration <configuring-ray>` page for more information.
 
+If you see ``Unable to connect to Redis. If the Redis instance is on a
+different machine, check that your firewall is configured properly.``,
+this means the ``--port`` is inaccessible at the given IP address (because, for
+example, the head node is not actually running, or you have the wrong IP
+address).
+
 If you see ``Ray runtime started.``, then the node successfully connected to
-the ``<address>``. If the ``<address>`` is inaccessible (because, for example,
-the head node is not actually running), then you will get an error such as
-``Unable to connect to Redis. If the Redis instance is on a different machine,
-check that your firewall is configured properly.``
+the IP address at the ``--port``. However, this does not necessarily mean that
+``ray.init(address='auto')`` will work. You can still get
+``Failed to connect to Redis, retrying.`` if there is a problem connecting to
+other ports.
```

Comment:
@dHannasch is there any note to add here about how to resolve this issue, ie. if there is a problem connecting to other ports, what would the next steps be?

Reply:
This could also just be "Raise an issue on Github"

Reply:
> @dHannasch is there any note to add here about how to resolve this issue, ie. if there is a problem connecting to other ports, what would the next steps be?

That's a good point.
I think the next steps would be to try to access the ports manually with something like `nc` or `nmap`. Does that sound sane?

Reply:
If that works for you then I think it sounds good! You know more about this than I do :)